src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -88,5 +91,28 @@ def generate_strategy_pool():
     pool.add_strategy(strategy.FORK_STRATEGY)
   if do_strategy(strategy.MUTATOR_PLUGIN_STRATEGY):
     pool.add_strategy(strategy.MUTATOR_PLUGIN_STRATEGY)
+  return pool
+
+
+def generate_strategy_pool():
+  """Generate a strategy pool based on probability
+  distribution from multi armed bandit experimentation."""
+  query = data_types.FuzzStrategyProbability.query()
+  distribution = list(ndb_utils.get_all_from_query(query))
+
+  # If we are not able to query properly, draw randomly according to
+  # probability parameters.
+  if not distribution:
```

Comment:
We also want to take this path if some environment variable isn't explicitly set.

Something like: if not distribution or not environment.get_value('USE_BANDIT_FOR_STRATEGIES'): (or any other name).